### PR TITLE
Add Latex doc build job to CI

### PR DESCRIPTION
**Pull request by inducer**
Targets #345.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -123,10 +123,12 @@ jobs:
             MINIFORGE_INSTALL_DIR=.miniforge3
             . "$MINIFORGE_INSTALL_DIR/bin/activate" testing
 
+            sudo aptitude install texlive
```
Comment (inducer) [inline .github/workflows/ci.yaml:126]:
```suggestion
            sudo apt-get install texlive
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -123,10 +123,12 @@ jobs:
             MINIFORGE_INSTALL_DIR=.miniforge3
             . "$MINIFORGE_INSTALL_DIR/bin/activate" testing
 
+            sudo apt-get install texlive
```
Comment (inducer) [inline .github/workflows/ci.yaml:126]:
```suggestion
            sudo apt-get install texlive latexmk
```

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -123,10 +123,12 @@ jobs:
             MINIFORGE_INSTALL_DIR=.miniforge3
             . "$MINIFORGE_INSTALL_DIR/bin/activate" testing
 
+            sudo apt-get install texlive latexmk
```
Comment (inducer) [inline .github/workflows/ci.yaml:126]:
```suggestion
            sudo apt-get install texlive-latex-extra latexmk
```

**Comment by inducer** [comment]
The built doc PDF is kind of fun to look at:
grafik

**Comment by inducer** [comment]
Builds locally now, I'll mark it ready for review.

**Review by MTCam — APPROVED** [review]
Excellent!  Thanks!

**Comment by inducer** [comment]
@MTCam This has conflicts now... could you take a look?

**Comment by inducer** [comment]
(And feel free to merge whenever it's convenient for you over on #345.)

**Pull request closed without merging**